db/migrate/20201118155951_add_creator_to_events.rb

```diff
@@ -0,0 +1,5 @@
+class AddCreatorToEvents < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :events, :user, index: true
+  end
+end
```

Comment:
- [ ] since you're adding `creator_id` to c`reate_events.rb` and assigning it as the `user_id` in the model, you don't need this file anymore